src/libs/ReportUtils.ts

```diff
@@ -2078,22 +2069,36 @@ function isReportFieldDisabled(report: OnyxEntry<Report>, reportField: OnyxEntry
 /**
  * Given a set of report fields, return the field of type formula
  */
-function getFormulaTypeReportField(reportFields: PolicyReportFields) {
+function getFormulaTypeReportField(reportFields: Record<string, PolicyReportField>) {
     return Object.values(reportFields).find((field) => field.type === 'formula');
```

Comment:
```suggestion
    return Object.values(reportFields).find((field) => field?.type === 'formula');
```
I got a crash on navigating to the LHN on desktop because I was getting a deleted field showing up as a null object. Disappeared after, so unsure what exactly triggered it, but seems safer to add this!